x-pack/plugins/code/public/components/admin_page/project_tab.tsx

```diff
@@ -134,10 +134,13 @@ class CodeProjectTab extends React.PureComponent<Props, State> {
   };
 
   public submitImportProject = () => {
-    this.props.importRepo(this.state.repoURL);
-    this.setState({ repoURL: '' });
+    if (!this.isURLInvalid(this.state.repoURL)) {
+      this.props.importRepo(this.state.repoURL);
+    }
   };
 
+  public isURLInvalid = (url: string) => url.trim() === '';
```

Comment:
this is a duplication here. let's import this method from the file above for the sake of future changes.